src/applications/caregivers/tests/e2e/fixtures/mocks/feature-toggles.json

```diff
@@ -2,10 +2,6 @@
   "data": {
     "type": "feature_toggles",
     "features": [
-      {
-        "name": "caregiverUseFacilitiesApi",
```

Comment:
This didn't seem to be used anywhere? These are mocked as a response from the backend, so the snake case is valid and the front end does the conversion.